explainaboard/processors/aspect_based_sentiment_classification.py

```diff
@@ -13,11 +13,11 @@
 from explainaboard.metrics.accuracy import AccuracyConfig
 from explainaboard.metrics.metric import MetricConfig
 from explainaboard.processors.processor import Processor
-from explainaboard.processors.processor_registry import register_processor
+from explainaboard.processors.processor_registry import processor_registry
 from explainaboard.utils.spacy_loader import get_named_entities
 
 
-@register_processor(TaskType.aspect_based_sentiment_classification)
+@processor_registry.register("aspect_based_sentiment_classification")
```

Comment:
Do you think this will make developers lose the convenience of the command prompt from IDE?

Reply:
@pfliu-nlp I'm not sure what is expected here, could you give us some examples you are concerned?

Reply:
@pfliu-nlp Thanks, it looks useful at this point.

But I think it'd be good to use just strings over an enum for task types for better library design, because it is a dynamic collection of strings, not a static information of the codebase over its lifetime.

If we or a third-party want to add a new task, they need to update this enum, and the change results in typing mismatch between forks of this repository. If they want to publish the task, we need to review/merge a new entry in TaskType every time and it causes unnecessary increase of the repository. The growth is also constrained by our load.

Reply:
@odashi, thanks for detailing this, which addresses my concern.